test/e2e/framework/network/utils.go

```diff
@@ -334,6 +339,44 @@ func (config *NetworkingTestConfig) GetEndpointsFromContainer(protocol, containe
 	return eps, nil
 }
 
+// GetResponsesFromTestContainer executes a curl via kubectl exec in a test container.
+func (config *NetworkingTestConfig) GetResponsesFromTestContainer(protocol, dialCommand, targetIP string, targetPort, tries int) []NetexecDialResponse {
+	ipPort := net.JoinHostPort(config.TestContainerPod.Status.PodIP, strconv.Itoa(testContainerHTTPPort))
+	// The current versions of curl included in CentOS and RHEL distros
```

Comment:
There is a lot of copy-pasting here from the `DialFromContainer`. Moreover, you don't seem to be calling this method with `tries` set to other value than `1`. Give that, I'd suggest the following. Get rid of the tries parameter here and change the return type to return a single `NetexecDialResponse` (also change a new of the method to singular -> `GetResponseFromTestContainer`). Then use this method in the `DialFromContainer` to implement a single try. It would make the code cleaner (less repetitions)

Reply:
I see you made the changes, thanks! Can we now call this method inside `DialFromContainer`?

Reply:
please see the latest commit